lib/src/utilities/simcompare.dart

```diff
@@ -144,19 +144,19 @@ abstract class SimCompare {
 
   /// Executes [vectors] against the Icarus Verilog simulator.
   static bool iverilogVector(
+    Module module,
     String generatedVerilog,
```

Comment:
I think we can remove `generatedVerilog` and call `generateSynth` within this function instead